challenge.js

```diff
@@ -9,6 +9,7 @@
  */
 
 // ANSWER
+  You would go to the web page, open developer tools and get the element ID of the item you want to insert the html element (parent) and use a command (unsure of the command) to add it as a child to the parent
```

Comment:
Yes, absolutely. Retrieving elements by ID only works for ones already attached to the DOM; frequently, we'll have a reference to an off-DOM element we created in our JS (via `document.createElement`) and need to attach it as a child to some on-DOM element. 1